twake/frontend/src/app/components/Leftbar/Channel/ChannelMenu.tsx

```diff
@@ -37,32 +88,49 @@ export default (props: Props): JSX.Element => {
     );
   };
 
-  const removeChannel = () => {};
+  const removeChannel = () => {
+    return channelsCollection.remove({ id: props.channel.id });
+  };
 
-  // Add translations
   let menu: object[] = [
     {
       type: 'menu',
       text: Languages.t('scenes.apps.messages.left_bar.stream.notifications'),
-      onClick: () => {},
       submenu: [
         {
           text: Languages.t('scenes.apps.messages.left_bar.stream.notifications.all'),
-          icon: 'check',
+          icon: props.channel.user_member?.notification_level === 'all' && 'check',
+          onClick: () => {
```

Comment:
Instead of using curly brackets (which means you need to return something) you could do this inline:

`(param: any) => //inline`

or
```js 
(param: any) => {
 return //something ;
}
```